packages/clerk-js/src/ui/utils/userUtils.ts

```diff
@@ -0,0 +1,3 @@
+export const userHasDefaultImage = (url: string) => {
```

Comment:
Lets rename to 
```suggestion
export const isDefaultAvatar = (url: string) => {
```
since the function has no knowledge of a "user"

Reply:
Also, lets use this one in `UserProfileAvatarUploader` as well

Reply:
`isDefaultProfileImage` is better as the terminology is about ProfileImage not Avatar.

Reply:
even better!

Reply:
@nikosdouvlis but it is under the file `userUtils` so maybe you are right🙃